### DISCUSSION TITLE
An NSFW filter ?

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#2733] [Q&A]
Downloading the detection model to /root/.ifnude/detector.onnx  .... this is a NSFW filter ????   

is it possible to turn it off ????

I do artistic nudes (no porn) and I'm a little afraid of this stuff...

**Comment by TheLastBen** [comment]
check which extension is downloading that model

**Comment by darth2602** [comment]
Roop
